.github/actions/prepare_FABulous_container/action.yaml

```diff
@@ -14,8 +14,8 @@ inputs:
     description: 'Additional python packages to install'
     required: false
     default: ''
-  install_GHDL_mcode:
-    description: 'Install GHDL with mcode backend'
+  install_GHDL:
```

Comment:
I think something with our rebasing went wrong or you had picked an old version of my commits.